src/algorithms/postburn/PostBurn.h

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Alex Jentsch, Jihee Kim, Brian Page
+//
+
+#include <DD4hep/Detector.h>
+#include <algorithms/algorithm.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <edm4eic/MCRecoParticleAssociationCollection.h>
+#include <edm4hep/MCParticleCollection.h>
+#include <spdlog/logger.h>
+#include <memory>
+#include <string>
+#include <string_view>
+
+#include "PostBurnConfig.h"
+#include "algorithms/interfaces/WithPodConfig.h"
+
+namespace eicrecon {
+
+  using PostBurnAlgorithm = algorithms::Algorithm<
+    algorithms::Input<
+      edm4hep::MCParticleCollection,
+      edm4eic::ReconstructedParticleCollection,
+          edm4eic::MCRecoParticleAssociationCollection
+    >,
+    algorithms::Output<
+      edm4hep::MCParticleCollection
+    >
+  >;
+
+  class PostBurn
+  : public PostBurnAlgorithm,
+    public WithPodConfig<PostBurnConfig> {
+
+  public:
+    PostBurn(std::string_view name)
+      : PostBurnAlgorithm{name,
+                            {"mcParticles", "ReconstructedParticleCollection", "MCRecoParticleAssociationCollection"},
+                            {"outputParticleCollection"},
+                            "Apply boosts and rotations to remove crossing angle and beam effects."} {}
+
+    void init(std::shared_ptr<spdlog::logger>& logger);
+    void process(const Input&, const Output&) const final;
+
+  private:
+
+    /** algorithm logger */
+    std::shared_ptr<spdlog::logger>   m_log;
```

Comment:
Not needed for algorithms that inherit from `algorithms::Algorithm`. See e.g. https://github.com/eic/EICrecon/pull/1483 for kind of changes needed related  to this.

Reply:
Done.